pkg/controller/security_group.go

```diff
@@ -415,16 +415,6 @@ func (c *Controller) syncSgLogicalPort(key string) error {
 	defer func() { _ = c.sgKeyMutex.UnlockKey(key) }()
 	klog.Infof("sync lsp for security group %s", key)
 
-	sg, err := c.sgsLister.Get(key)
```

Comment:
why touch this？

Reply:
We should assign the variable near the place where it is used.